### PR TITLE
Updated number of icons in the readme

### PR DESCRIPTION
**Pull request by DenverCoder1**
The number in the readme is "Over 1500."

It is more accurate to say "Over 2000" since Simple-icons has 2020+ icons now! :tada:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ echo file_get_contents('path/to/package/icons/simpleicons.svg');
 ?>
 ```
 
-## Third Party Extensions
+## Third-Party Extensions
```
Comment (adamrusted) [inline README.md:141]: I don't think this hyphen needs adding, though there are uses of both variations across the internet. Thoughts, @simple-icons/maintainers?

**Comment by adamrusted** [comment]
Wow, that 2k came around quick! 🤯

File: README.md (hunk 1 above)
Comment (DenverCoder1) [reply README.md:141]: I believe it's meant to be hyphenated when used as an adjective, but can be with a space when used as a noun.

So grammatically speaking, this usage would be better with a hyphen. If you'd rather I not change it, I can put it back though.

Comment (adamrusted) [reply README.md:141]: Figured it'd be something like that, though the use cases of the phrase I find seem to mix and match between the two, hence why I've pinged the other maintainers to gain their thoughts on it. If there's no objections by the end of the day, I'll look to merge this one in.

**Review by mondeja — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mondeja) [reply README.md:141]: I can see a lot of references about hyphenation of compound words when used as adjetves. Altough I'm not a native english speaker, this change seems good to me.

**Review by sachinraja — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (adamrusted) [reply README.md:141]: Marking this as resolved, and merging this PR due to multiple other maintainers approving. Thanks for your work @DenverCoder1

**Pull request merged**